test/conformance/testing/include/uur/fixtures.h

```diff
@@ -22,14 +22,14 @@
     }                                                                          \
     (void)0
 
-#define UUR_ASSERT_SUCCESS_OR_UNSUPPORTED(ret)                                 \
+#define UUR_ASSERT_SUCCESS_OR_UNSUPPORTED(ret) do {                            \
```

Comment:
I guess because `status` wasn't enclosed in any scope it was leaking and causing the errors elsewhere? Could the macro just be enclosed in `{ }` instead of a loop maybe?

Reply:
Yep thanks for the link!